packages/dcos-checks/extra/dcos-checks/cmd/binaries_test.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import "testing"
+
+// TestBinariesCheckBinaryExists validates binary exists
+func TestBinariesCheckBinaryExists(t *testing.T) {
+	c := &BinariesCheck{
+		Name: "TEST",
+	}
+	mockCLICfg := &CLIConfigFlags{
+		NodeIPStr: "127.0.0.1",
+		Role:      "master",
+		ForceTLS:  false,
+	}
+	err := c.binaryExists(mockCLICfg)
```

Comment:
can we make this unittest more reliable? if i run the unittest on my mac it will fail because i don't have some binaries installed, we should probably test function logic instead of implementation.